monkey/infection_monkey/main.py

```diff
@@ -32,12 +32,14 @@ class OTPFormatter(logging.Formatter):
     Formatter that replaces OTPs in log messages with asterisks
     """
 
-    def format(self, record):
-        otp_regex = re.compile(f"{AGENT_OTP_ENVIRONMENT_VARIABLE}=[a-zA-Z0-9]*")
-        otp_replacement = f"{AGENT_OTP_ENVIRONMENT_VARIABLE}={'*' * 6}"
+    OTP_REGEX = re.compile(f"{AGENT_OTP_ENVIRONMENT_VARIABLE}=\\S+\\s")
```

Comment:
This will work if the `OTP_VAR=<otp>` is followed by whitespace. It looks like all our commands currently follow it up with whitespace. Do you foresee any instance where this would not be the case?

Reply:
That's a good thought. It should always be the case. On Linux, a space is required in order to separate the variable from the command. On Windows, I think we often use a semicolon, so I've modified this to be more robust.